cmd/entrypoint/main.go

```diff
@@ -59,6 +59,8 @@ var (
 	enableSpire            = flag.Bool("enable_spire", false, "If specified by configmap, this enables spire signing and verification")
 	socketPath             = flag.String("spire_socket_path", "unix:///spiffe-workload-api/spire-agent.sock", "Experimental: The SPIRE agent socket for SPIFFE workload API.")
 	resultExtractionMethod = flag.String("result_from", featureFlags.ResultExtractionMethodTerminationMessage, "The method using which to extract results from tasks. Default is using the termination message.")
+	stepName               = flag.String("step_name", "", "Name of the step")
+	stepResults            = flag.String("step_results", "", "step results if specified")
```

Comment:
This needs more details on the expected format, data, etc.

Reply:
Coming back after reading the impl - I don't think you want this to be JSON encoded paths. You should be able to deterministically determine result paths from a list of names (e.g. `path := filepath.Join("%s/results/%s", stepMetadataDir, name)`)

Just as there's a finite size to output data, there is also a finite size for how much data we can shove into command line args, so we want to try and minimize this if we can.

Reply:
sounds good. I made it consistent with results.